assets/css/style.css

```diff
@@ -406,7 +406,7 @@ html[dir='rtl'] .pull-left:not(#rtlChecker) { /* sass-lint:disable-line class-na
     float: right !important; /* sass-lint:disable-line no-important */
 }
 
-/* D3 path chooser stuff */
+/* translation path chooser stuff */
```

Comment:
The "choose" terminology is used throughout my code: "chooser", "chosen", etc. If there is some specific reason you prefer "select" over "choose", I'll go through everything and change it.

Reply:
My primary concern is that the word "chooser" is somewhat awkward in (at least American) English.

e.g.

![image](https://user-images.githubusercontent.com/4211838/33030010-22406426-cde8-11e7-992a-17189023f976.png)

![image](https://user-images.githubusercontent.com/4211838/33030020-291c13b2-cde8-11e7-94b5-2e3c93f09328.png)